docs/Project.toml

```diff
@@ -9,5 +9,6 @@ IntervalRootFinding = "d2bf35a9-74e0-55ec-b149-d360ff49b807"
 IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Polynomials = "f27b6e38-b328-58d1-80ce-0feddd5e7a45"
+Roots = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
```

Comment:
Thanks, should this change go into "docs/make.jl"? Then it would persist.

Reply:
> This is actually a Julia 1.3 requirement rather than a Documenter requirement.
>
> _Originally posted by `@omus` in https://github.com/JuliaCI/PkgTemplates.jl/issues/139#issuecomment-590444146_

`Pkg.develop` adds this dependency automatically, so not adding `Roots` doesn't affect the ci.
However, when people want to build the documentation locally, they may encounter some errors.

Reply:
Okay, I see. Thanks.